src/ucp/wireup/select.c

```diff
@@ -1295,7 +1295,7 @@ ucp_wireup_mem_reg_cost(ucp_context_t *context,
     }
 
     if (remote_addr->dst_version > UCP_RELEASE_LEGACY) {
-        return UCP_RCACHE_LOOKUP_FUNC;
```

Comment:
It seems that it shouldn't be configurable, to avoid wire compatibility issues.

@brminich